torch/csrc/generic/methods/TensorRandom.cwrap

```diff
@@ -73,29 +82,33 @@ static void THTensor_(random1__)(THTensor *self, THGenerator *gen, long b)
 
 [[
   name: uniform_
-  defined_if: defined(TH_REAL_IS_FLOAT) || defined(TH_REAL_IS_DOUBLE)
+  types:
+    - floating_point
+  processors:
+    - CPU
+    - CUDA
   cname: uniform
   return: self
   arguments:
     - THTensor* self
     - arg: THGenerator* generator
       default: THPDefaultGenerator->cdata
       kwarg_only: True
-    - arg: real from
+    - arg: double from
```

Comment:
Is there are particular reason to pin it to double, or does it just make the processing simpler?